cove_360/lib/threesixtygiving.py

```diff
@@ -979,6 +990,180 @@ def process(self, grant, path_prefix):
         self.message = mark_safe(self.check_text['message'][self.grants_percentage])
 
 
+class ImpossibleDates(AdditionalTest):
+    """Check if dates supplied are plausible (eg no 31st Feb)"""
```

Comment:
Let's also say here that we check for dates that are plausible but didn't happen (eg 29th of Feb in a non-leap year) because this message is used by us and Katherine to remember what these checks do.

Reply:
I will change it to 

Check if dates supplied are plausible (eg no 31st Feb) or are plausible but didn't happen (eg 29th of Feb in a non-leap year).

Reply:
Sounds good!